### ISSUE TITLE
Same API request, different dataset each time (decimals in floats are different each time)!

### ISSUE THREAD

**Issue by teneon** [ranaroussi/yfinance#1057]
Hi there,

i have noticed a weird issue. Each request returns slightly different data.

import yfinance as yf
df = yf.Ticker("UNH")
df = df.history(period="max")
df.to_csv("df.csv")

For demostration purpose i have downloaded this same stock 3 times (one download per python execution) and named the three files as 'df.csv', 'df1.csv', 'df2.csv'. All files are slightly different.  Basically CRC and even filesize is different for all of them.

Let's analyze it with 'cksum' command. The first number is CRC sum and the second number is file size.

**cksum df.csv**
3599934556 947002 df.csv

**cksum df1.csv**
4145465075 946595 df1.csv

**cksum df2.csv**
3647012562 946461 df2.csv

This is IMPOSSIBLE i thought... and then i started to dig deeper. I compared df.csv and df1.csv with 'diff' command
**diff df.csv df1.csv**

....and there are a LOT of differences in both files. Basically i figured out then that the problem is in floats. For some reason floats are slightly different each time you send a new request to the API.

For example in my case:
**df1.csv: 2022-03-08,475.69087992904736, ...
df2.csv: 2022-03-08,475.69091079082295, ...**

And it is not a problem with saving CSV,  numbers are different each time data is returned from API, even before saving to .csv. I saved it to csv, just so i could analyse it easier and show it to you. 

Any ideas what's happening here?

Best regards,
Jim

**Comment by ValueRaider** [comment]
Please look through recent issues first before submitting

https://github.com/ranaroussi/yfinance/issues/1038

**Comment by teneon** [comment]
Hi @ValueRaider ,

thanks for your quick reply. Sorry about that. I did browse through several related issues. I also saw issue #1038 ,the one you have posted in your reply. BUT, if i understand correctly, the user in the related issue  is talking about **adjusted** close price issue?

You have commented there:

> _The prices are exactly the same. The adjusted price is changing, and in a consistently negligible way so first thought goes to floating-point rounding errors. This is possible depending on how adjustment calculation is structured._

What i have observed is that ALL float numbers are changing, open, high, low, close.
Any thoughts? 

Best regards,
Jim
